release-notes/swan-lake-alpha5-release-note.md

```diff
@@ -37,8 +37,62 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
+
+##### `lang.xml:getDescendants()` Lang Library Function
+
+The `lang.xml:getDescendants()` lang library function has been introduced to retrieve a sequence of the descendant elements for a given element, in document order.
+
+##### Bitwise Operations as Constant Expressions
+
+Bitwise binary operations `<<`, `>>`, `>>>`, `^`, `&`, `|` are now allowed as constant expressions.
+
+```ballerina
+const BASE = 0x10;
+const int SHIFTED = (BASE << 2) | 0xa;
+```
+
 #### Improvements
 
+##### Improvements to the XML Literal
+
+XML template literals now support the inclusion of multiple XML items of the same `xml` subtype as well as different `xml` subtypes.
+
+```ballerina
+xml x1 = xml `<ele>item</ele><!--comment-->text<?foo?>`;
+xml<xml:Element> x2 = xml `<ele>item1</ele><ele>item2</ele>`;
+xml<xml:Comment> x3 = xml `<!--comment1--><!--comment2-->`;
+xml<xml:ProcessingInstruction> x4 = xml `<?foo?><?bar?>`;
+string interpolation = "Interpolation";
+xml<xml:Text> x5 = xml `text ${interpolation}`;
+```
+
+##### Normalization of XML Attribute Values
+In XML template literals, character references within XML attribute values are now normalized to the referring character. For example, the attribute value `e.attr` in the following example is now `a&b`.
+
+```ballerina
+xml:Element e = xml `<elem attr="a&amp;b" />`;
+```
+
+##### Breaking Changes
+- The `lang.array:lastIndexOf` and `lang.array:indexOf` lang library functions can only be used with arrays of which the element types are subtypes of the `anydata` type.
+- A compilation error occurs if the field visibility qualifiers in the including type and the included type with object type inclusion are not the same.
+- Remote method call action cannot be used to call non-remote methods.
+- The `xml` and `readonly` type-descriptors can no longer be used as expressions. A reference to a type definition defined with such type-descriptors needs to be used instead.
+- The argument passed to the second parameter of the `attach` method of the listener for service declarations has been fixed as follows. If there is no attach point in the service `()` (nil) is passed as the argument. If the attach point is the root resource path (“/”) an empty array is passed.
+- The `lang.xml:getContent()` function no longer accepts `xml:Text` as an argument. The new `lang.xml:data()` function can be used instead.
+- `lang.value` functions cannot be called on expressions of type `object`. Function call syntax (`value:func(obj)`) can be used instead.
```

Comment:
```suggestion
- The `lang.value` function cannot be called on expressions of type `object`. The function call syntax (`value:func(obj)`) can be used instead.
```

Reply:
`lang.value` is a library with many functions.